NGitLab/Models/IssueType.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace NGitLab.Models
+{
+    public enum IssueType
+    {
+        issue,
```

Comment:
Please use PascalCase names.

PascalCase <-> snake_case serialization should work correctly when there is only 1 word. This means you'll need to provide  `[EnumMember(Value = "test_case")]` attribute for `TestCase`.

Reply:
NVM, what I said is only partially right and only for JSON-serialized `enum`s. Here we're dealing with query params...